salt/utils/profile.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
Several of these new utils files don't have docstrings at the top. We should probably add at least something simple here in these files.